Added Emoji Support :clap: :fire: :clap: #225

itsWill opened the pull request:
💥 Emojis are here! 💥

I've added some tests to keep it 💯

This is implemented using the ⚡ gemoji gem ⚡

I've also made all titles titleized but don't know if that is always appropriate ❓ ❔ ❓

☀️ emojineers at 🚜 ☀️

New PR to fix 📛  tests

sophiedeziel commented on app/helpers/emoji_helper.rb:
```diff
@@ -0,0 +1,12 @@
+    h(content).to_str.gsub(/:([\w+\s-]+):/) do |match|
+      name = $1.tr(" ", "_").downcase
+      if emoji = Emoji.find_by_alias(name)
+        %(<img alt="#{name}" src="#{image_path("emoji/#{emoji.image_filename}")}" style="vertical-align:middle" width="20" height="20" />)
```
A css class is prefered. Remove the inline style. Also, width and height should be set in `em` to scale with font-size.

sophiedeziel commented on app/views/events/_event.html.erb:
```diff
@@ -1,11 +1,11 @@
   </div>
   <div class="markdown-body">
-    <%= render_markdown_as_html event.body %>
+    <%= render_markdown_as_html emojify(event.body) %>
```
The emojify helper should wrap `render_markdown_as_html`, not the opposite. Emojify will generate html tags in a text formatted with markdown.

itsWill replied:
I actually think makes sense to include the `emojify` helper within the `render_markdown_as_html` helper for two reasons:  

1. The way I see it github emoji i.e colon emoji colon syntax is part of the github markdown so it should be treated as such.
2.  Calling the `emojify` method shouldn't result in markdown being outputted, only the replacement of the :emoji: syntax with the right emoji, that doesn't seem like expected behavior.

 I see the `emojify`method as a helper method to the `render_markdown_as_html` method which replaces "markdown emoji syntax" with the right emoji.

sophiedeziel replied:
Yes!

sophiedeziel commented on app/helpers/emoji_helper.rb:
```diff
@@ -0,0 +1,12 @@
+module EmojiHelper
+  def emojify(content)
```
Add some rspec specs to test that feature.

sophiedeziel commented:
There's a bunch of tools you can use before commiting or pushing. `rubocop -a` will auto-fix most things hound complains about. I love it 😸

itsWill commented:
Refactored to add `emojify` method as part of the `render_markdown_as_html` method.

 Added `byebug` to the `Gemfile` for those of us who use it instead of `pry`. I've had to update the `.gitignore` as well.
